code/src/index.css

```diff
@@ -1,13 +1,210 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,200;0,400;0,500;0,600;0,700;0,800;1,400&display=swap');
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+	background-color: black;
+	margin: 0;
+	font-family: 'Montserrat', sans-serif;
+	/* font-family: -apple-system, BlinkMacSystemFont, "Montserrat", "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+    sans-serif; */
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+	box-sizing: border-box;
+}
+
+/* code {
+	font-family: source-code-pro, Menlo, Monaco, Consolas, 'Montserrat',
+		'Courier New', monospace;
+} */
+
+.poster-img {
+	width: 100%;
+	height: 100%;
+	display: inherit;
+	object-fit: cover;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.movie-list-container {
+	margin: 0 auto;
+}
+
+.movie {
+	position: relative;
+	display: inherit;
+}
+
+.overlay-text {
+	color: white;
+	padding: 2rem;
+	transition: 0.3s ease;
+	font-size: 1.2em;
+}
+
+.overlay-text p {
+	font-size: 1rem;
+}
+
+.detail-poster {
+	width: 90%;
+	max-width: 25rem;
+	box-shadow: 0rem 0rem 2rem 1rem rgba(0, 0, 0, 0.521);
+}
+
+.detail-text {
+	font-size: 1.15rem;
+	line-height: 1.6;
+	max-width: 55em;
+	width: 90%;
+}
+
+.backdrop {
+	z-index: -1;
+	width: 100%;
+	height: 100%;
+}
```

Comment:
Cool using z-index. It's rare I see z-index I feel being used. And it's so useful.